metaci/release/admin.py

```diff
@@ -10,6 +11,29 @@ class ImplementationStepInline(admin.TabularInline):
 
 @admin.register(Release)
 class ReleaseAdmin(admin.ModelAdmin):
+    def get_form(self, request, obj=None, **kwargs):
+        form = super(ReleaseAdmin, self).get_form(request, obj, **kwargs)
+        repo_id = request.GET.get("repo_id")
+        if repo_id is not None:
+            repo = Repository.objects.get(pk=int(repo_id))
+            form.base_fields["repo"].initial = repo
+            if repo.latest_release:
+                if len(repo.latest_release.version_number.split(".")) > 1:
+                    next_version = f"{repo.latest_release.version_number.split('.')[0]}.{int(repo.latest_release.version_number.split('.')[1]) + 1}"
+                    form.base_fields["version_name"].initial = next_version
+                    form.base_fields["version_number"].initial = next_version
+                    form.base_fields[
+                        "git_tag"
+                    ].initial = f"{repo.release_tag_regex}{next_version}"
+            else:
+                form.base_fields["version_name"].initial = "1.0"
+                form.base_fields["version_number"].intial = "1.0"
+                form.base_fields[
+                    "git_tag"
+                ].initial = f"{repo.release_tag_regex}{next_version}"
```

Comment:
@jofsky I think we want something like this to get the version from the most recent beta release:

    # builds default to sorting with the most recent first
    latest_beta_tag = repo.builds.filter(plan__role="beta_test").first().branch.name
    # latest_beta_tag is something like "tag: uat/3.198-Beta_4"
    version = re.search('([\d\.]+)-Beta_\d+$', tag).group(1)

Reply:
@jofsky Now I'm having second thoughts about using betas, because it's possible that there might not yet be a new beta since the last final release at the point when the new release is planned. So let's go with your existing approach here.